general/bundler_imports.md

```diff
@@ -0,0 +1,160 @@
+# Bundler Imports
+
+This document is aimed at explaining how npm bundler imports work.
+
+## Why do we need npm bundler imports
+
+As all of you probably know, usually JavaScript web applications need a bundling phase to convert npm modules into a single `.js` file that can be loaded and executed by browsers. The reference tool for this task is [webpack](https://webpack.js.org/).
```

Comment:
Pedantic nit: it's probably "most popular" rather than "reference" — as a crude measure, npm weekly stats:

- webpack: 12.3M
- rollup: 2.3M
- browserify: 1M
- parcel: 59K
- esbuild: 44K
- microbundle: 20.7K
- fuse-box: 3.7K
- liferay-npm-bundler 😁: 2.6k

If webpack were "reference", you'd expect there to be lots of copies of webpack out there, but this list shows a number of widely used alternatives, and most of these exist precisely because they wanted to take a _different_ approach.

Reply:
O_o  12.3M (webpack) vs 3.4M (rest) => 78% market share

But yeah... Probably not reference :sweat_smile: as the other are alternatives, not clones. 

However, what I meant with reference is that the "bundling standard" (if such standard exists :roll_eyes:) has been created by webpack, and the rest of the bundlers, even though they work differently, almost always implement at least webpack functionality. Everybody expects webpack quirks to work when using an alternative bundler (one example if the `browser` field of `package.json`, but there are more in the way you write the code).

Reply:
I have rephrased it to:

> The most popular tool (and the one that developers usually target) for this task is [...]